tools/create_data.py

```diff
@@ -274,12 +300,23 @@ def semantickitti_data_prep(info_prefix, out_dir):
 parser.add_argument(
     '--only-gt-database',
     action='store_true',
-    help='Whether to only generate ground truth database.')
+    help='''Whether to only generate ground truth database.
```

Comment:
It's better to illustrate different situations in the corresponding documents.

Reply:
For example, for different tasks, we need different parameter settings for data info collection